example-app/src/main/java/ai/promoted/RestaurantListActivity.kt

```diff
@@ -0,0 +1,137 @@
+package ai.promoted
+
+import ai.promoted.dummy.DummyContent
+import ai.promoted.proto.event.ActionType
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
+import androidx.core.widget.NestedScrollView
+import androidx.recyclerview.widget.RecyclerView
+
+/**
+ * An activity representing a list of Pings. This activity
```

Comment:
What is a `Ping`?

Reply:
😳 no clue--must've been carry-over from the IDE template wizard when creating the screens. Fixed in latest commit.